tools/src/main/scala/org/apache/spark/sql/rapids/tool/profiling/EventsProcessor.scala

```diff
@@ -404,8 +404,8 @@ class EventsProcessor(app: ApplicationInfo) extends EventProcessorBase[Applicati
         app.accumIdToStageId.put(res._2.id, event.stageInfo.stageId)
         arrBuf += thisMetric
       } catch {
-        case e: ClassCastException =>
-          logWarning("ClassCastException when parsing accumulables for task " +
+        case e: Exception =>
```

Comment:
should we catch exceptions more exhaustively with matching on `Throwable`?

Reply:
if it's an Error, I think we just want to fail as that could be something more than just related to the accumulable, what I actually probably should do is make this match on NonFatal(e)

Reply:
+1 for NonFatal, it's consistent with similar usages in our code base